app/celery/letters_pdf_tasks.py

```diff
@@ -128,21 +130,30 @@ def update_validation_failed_for_templated_letter(self, notification_id, page_co
 
 @notify_celery.task(name="collate-letter-pdfs-to-be-sent")
 @cronitor("collate-letter-pdfs-to-be-sent")
-def collate_letter_pdfs_to_be_sent():
+def collate_letter_pdfs_to_be_sent(print_run_deadline_utc=None):
```

Comment:
Good call with the `_utc` suffix - I would rather we start being this explicit more often 👍